.github/CODEOWNERS

```
@@ -1,2 +1,3 @@
 * @afonsojramos
-* @bmulholland
\ No newline at end of file
+* @bmulholland
+* @setchy
```

I think this needs to be:
```suggestion
* @afonsojramos @bmulholland @setchy
```

---

Exactly, thought it might be something like that! From the [example](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) it looks like that is, indeed, the way.